llvm/reducertest/intsummod.ll

```diff
@@ -0,0 +1,430 @@
+ivate unnamed_addr constant [26 x i8] c"Result: %d/%d successes!\0A\00", align 1
```

Comment:
Looks like the example got cut off.

Reply:
Interesting...  It should have done a straight copy -- I'll double check it again...